recipes/r-shogun/meta.yaml

```diff
@@ -0,0 +1,149 @@
+{% set name = "r-shogun" %}
+
+{% set version = "6.1.3" %}
+{% set commit = 'shogun_' + version %}
+{% set sha256 = "75f4d555efe06eaa7c4c12a1dc942f6e4d41a8ed495777a790b9bd9df936c19c" %}
+
+{% set build = 0 %}
+
+# Shogun doesn't produce a single tarball you can build from anymore;
+# need to get the gpl-shogun tarball too. Remember to update this when updating
+# the version!
+{% set gpl_commit = "29ca2ff930cd5d963b3a78e79160cc04d48970c2" %}
+{% set gpl_sha256 = "ff8a46b7775e784f07e9ffb02e7cd9b101f508c1c7b7d8cdbf1024a2efacdb22" %}
+{% set gpl_url = "http://github.com/shogun-toolbox/shogun-gpl/archive/" + gpl_commit + ".tar.gz" %}
+
+{% set blas_variant = "openblas" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: shogun_{{ version }}.tar.gz
+  url: https://github.com/shogun-toolbox/shogun/archive/{{ commit }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: {{ build }}
+  features:
+    - blas_{{ blas_variant }}  # [not win]
+  skip:
+    - true  # [win32]         # shogun doesn't build 32-bit
+    - true  # [win and py2k]  # shogun doesn't build on VC9
+    - true  # [py!=36]        # python just build dep, don't need matrix
+  script: |
```

Comment:
can you please put this script into a separate build.sh

Reply:
@bgruening Unfortunately, not really, because I need the `gpl_commit` and `gpl_sha256` Jinja values in the build script for `get_tarball.py`, and if you have both a `build.sh` and a `build/script` then it runs the `build.sh` first.

I guess I could pull out the build into a different script called with arguments in `build/script`, but that doesn't really seem cleaner....